core/src/main/scala/kafka/server/ZkAdminManager.scala

```diff
@@ -871,7 +872,7 @@ class ZkAdminManager(val config: KafkaConfig,
         users.get.filterNot(usersToSkip.contains).foreach { user =>
           try {
             val userConfigs = adminZkClient.fetchEntityConfig(ConfigType.USER, Sanitizer.sanitize(user))
-            addToResultsIfHasScramCredential(user, userConfigs, true)
+            addToResultsIfHasScramCredential(user, userConfigs, explicitUser = true)
           } catch {
             case e: Exception => {
```

Comment:
the braces here is redundant not sure if we want to clean this up as well or not

Reply:
We have this in few places in this class as well

Reply:
Yeah in Scala braces are not required around multi-line blocks. I've not made this change because braces are required in Java and we have the braces in Scala all over the code base. Changing this is probably a >500 line diff.

Reply:
ough >500 is too much. Well they will get cleanup as we move more scala to java :D

Reply:
That's just my guesstimate, but I expect it to be large. I kind of focused on the low hanging fruits here

Reply:
make sense don't worry about it. We are moving more and more from scala to java anyway so these will get resolved over time.